ucsmsdk/ucssession.py

```diff
@@ -408,8 +408,9 @@ def __validate_connection(self):
                                             dn=top_system.dn)
                 response = self.post_elem(elem)
                 if response.error_code != 0:
-                    raise UcsException(response.error_code,
-                                       response.error_descr)
+                    # raise UcsException(response.error_code,
+                    #                    response.error_descr)
```

Comment:
@ragupta-git yep, agree with bdemers. better removed than commented. And add a test case for the forceflag.